### ISSUE TITLE
Try it out! button not working on IE11 and Edge

### ISSUE THREAD

**Issue by SWSAmor** [DapperDox/dapperdox#65]
As the subject. We tried on IE 11.483.15063.0 and Edge 40.15063.0.0

**Comment by zxchris** [comment]
Yeah, that's not right, we'll spin up something with ie11 and look into that. Thanks for the report.
